src/python/gem5/simulate/simulator.py

```diff
@@ -185,6 +190,18 @@ def unique_exit_event():
 
         self._checkpoint_path = checkpoint_path
 
+    def schedule_simpoint(
+        self,
+        simpoint_end_inst: int = 0,
+        simpoint_start_insts: List[int] = []
+        ):
+        self._board.get_processor().get_cores()[0].\
+                        set_simpoint(simpoint_end_inst,simpoint_start_insts)
+
+    def schedule_simpoint_restore_stop(self, simpoint_interval: int):
```

Comment:
What is this function used for? Could you add pydoc for this and the function above?